presentations/2023-08-02_mlcsu-ksn-meeting/index.qmd

```diff
@@ -0,0 +1,165 @@
+---
+title: "Travels with R and Python - the power of data science in healthcare"
+subtitle: "[Chris Beeley](mailto:chris.beeley1@nhs.net)"
+format:
+  revealjs:
+    theme: [default, ./su_presentation.scss]
+    transition: none
+    chalkboard:
+      buttons: false
+    preview-links: auto
+    logo: logo_yellow.svg
+    slide-number: false
+    auto-animate: true
+---
+
+[ds_presentations]: https://the-strategy-unit.github.io/data_science/presentations/2023-08-02_mlcsu-ksn-meeting
+
+## What is data science?
+
+* "A data scientist knows more about computer science than the average statistician, and more about statistics than the average computer scientist"
+
+## Drew Conway's famous Venn diagram
+
+<img src="Data_Science_VD.png" alt="Data science Venn diagram"></a>
+
+[Source](http://drewconway.com/zia/2013/3/26/the-data-science-venn-diagram)
+
+## What are the skills of data science?
+
+* Analysis
+  * ML
+  * Stats
+  * Data viz
+* Software engineering
+  * Programming
+  * SQL/ data
+  * DevOps
+  * RAP
+
+## What are the skills of data science?
+  
+* Domain knowledge
+  * Communication
+  * Problem formulation
+  * Dashboards and reports
+
+## Stats and data viz
+
+* ML leans a bit more towards atheoretical prediction
+* Stats leans a bit more towards inference (but they both do both)
+* Data scientists may use different visualisations
+  * Interactive web based tools
+  * Dashboard based visualisers e.g. [{stminsights}](https://github.com/cschwem2er/stminsights)
```

Comment:
I've started linking to the first header in READMEs as I'm not sure people are always that familiar with GitHub and miss the reason for the link https://github.com/cschwem2er/stminsights#stminsights or the pkddown https://cschwem2er.github.io/stminsights/